### ISSUE TITLE
flexibility of specifying ScanPlan and detector type

### ISSUE THREAD

**Issue by chiahaoliu** [xpdAcq/mission-control#109]
Another important feature is the use of a control file, probably the sample spreadsheet itself, which says :
-        Sample X, run PDF at room  temperature

-        Sample Y, run a tseries with XRD and PDF combined

-        Sample Z, run room temperature XRD and a PDF Tramp

**Comment by sbillinge** [comment]
mother of all spreadsheets:
sheet1 = sample spreadsheet (as now)
sheet2 = scan spreadsheet
sheet3 = magazine spreadsheet:   scan_number, magazine_position, sample_number, scanPlan_number

**Comment by sbillinge** [comment]
@dooryhee are you ok if this goes to 2019Cycle1 release.

**Comment by dooryhee** [comment]
Yes. We already have an xpdAcq solution that works, with a compact (and very flexible) xrun syntax (thanks CJ!).
E
